apps/doji-frontend/src/modules/chat/components/ChatRoomDialog/index.tsx

```diff
@@ -95,16 +93,20 @@ export const ChatRoomDialog: React.FC<ChatRoomDialogProps> = (props) => {
           <Typography variant="subtitle2" color="ink.lighter" fontWeight={400} mb={2}>
             Select the user to chat with
           </Typography>
-          <Select fullWidth {...register('username', { value: null })}>
-            {options.map((name) => (
-              <MenuItem key={name.value} value={name.value}>
-                {name.label}
-              </MenuItem>
-            ))}
-          </Select>
+          <Autocomplete
+            disablePortal
+            id="combo-box-demo"
+            options={options}
+            size="small"
+            loading={isLoading}
+            loadingText="Loading..."
+            renderInput={(params) => (
+              <TextField {...params} {...register('username')} label="username" fullWidth />
+            )}
+          />
         </DialogContent>
         <DialogActions>
-          <Button variant="outlined" size="small" onClick={props.close}>
+          <Button variant="outlined" size="small">
```

Comment:
WHY?

Reply:
Reset already with the file from branch main